sass/main.scss

```diff
@@ -423,11 +423,25 @@ footer {
     }
 }
 
+@media screen and (max-width: 1440px) {
+    #hero {    
+        #bh7 {
+            margin-top: -217px;
```

Comment:
Arbitrary number here -- could you use something more like `-200px`? 
However this still clips on my 16" mac, making the window sightly narrower.